shell/platform/linux/fl_renderable.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_LINUX_FL_RENDERABLE_H_
+#define FLUTTER_SHELL_PLATFORM_LINUX_FL_RENDERABLE_H_
+
+#include <gdk/gdk.h>
+
+#include "flutter/shell/platform/embedder/embedder.h"
+
+G_BEGIN_DECLS
+
+G_DECLARE_INTERFACE(FlRenderable, fl_renderable, FL, RENDERABLE, GObject);
+
+/**
+ * FlRenderable:
+ *
+ * An interface for a class that can render views from #FlRenderer.
+ *
+ * This interface is typically implemented by #FlView.
+ */
+
+struct _FlRenderableInterface {
+  GTypeInterface g_iface;
+
+  void (*redraw)(FlRenderable* renderable);
+};
+
+/**
+ * fl_renderable_redraw:
+ * @renderable: an #FlRenderable
+ *
+ * Indicate the renderable needs to redraw.
```

Comment:
This wording makes it sound like the renderable can return before completing the redraw. Is that so?

If yes, consider adding a sentence like: `The renderable can return before the redraw has completed`.

If not, consider tweaking the sentence to something like: `Redraw the renderable.`